vm/runtime.ts

```diff
@@ -1913,8 +1897,8 @@ module J2ME {
     }
   }
 
-  export function checkDivideByZeroLong(value: Long) {
-    if (value.isZero()) {
+  export function checkDivideByZeroLong(low: number, high: number) {
```

Comment:
BaselineCompiler.emitDivideByZeroCheck will need updating to reflect this change. I guess it doesn't need to happen in conjunction with this change, since the compiler doesn't work at the moment. :-)